### PR TITLE
Add circular reference back to marshmallow.md

### PR DESCRIPTION
**Pull request by chriswengermusic**

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/pinch/go-out/follow-rabbit/follow-rabbit.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 You jumped into the hole and slide down for quite some time, until you hit something soft.
 
-In the dim light, you try to stand and see where you are.
\ No newline at end of file
+In the dim light, you try to stand and see where you are.
+
+When your eyes adjust to the ambient light, you realize you're back in the 
+
+[marshmallow room](english/marshmallow.md).
```
Comment (casey-collab) [inline english/pinch/go-out/follow-rabbit/follow-rabbit.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by chriswengermusic** [comment]
I've tried several things, and I can't get the link to work. I initially omitted ".." before the path, but even with that on, it is still broken. How do I link back to "..english/marshmallow.md" if not like that?

**Comment by cbuckey-uda** [comment]
I believe you will need to use `..` three times, since you are three directories down from the `english` directory. So try `../../../english/marshmallow.md`.

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look.

File: english/pinch/go-out/follow-rabbit/follow-rabbit.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
 You jumped into the hole and slide down for quite some time, until you hit something soft.
 
-In the dim light, you try to stand and see where you are.
\ No newline at end of file
+In the dim light, you try to stand and see where you are.
+
+When your eyes adjust to the ambient light, you realize you're back in the 
+
+[marshmallow room](../../../english/marshmallow.md).
```
Comment (casey-collab) [inline english/pinch/go-out/follow-rabbit/follow-rabbit.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Sorry, going up three directories puts you in the english directory, so it would actually be `../../../marshmallow.md`.

**Comment by chriswengermusic** [comment]
Ok, NOW it's good. Caroline, thanks for your help, and I'm sorry it took me so long. I really appreciated this course, too.

**Comment by cbuckey-uda** [comment]
You're welcome! Glad you enjoyed the course! @casey-collab, please take another look.

**Comment by casey-collab** [comment]
Looks great! :four_leaf_clover:

**Pull request merged**